lib/schema.js

```diff
@@ -1504,6 +1504,49 @@ Schema.prototype.remove = function(path) {
   }
 };
 
+/**
+ * Loads an ES6 class into a schema. Maps setters + getters, static methods, and instance methods to schema virtuals, statics, and methods.
+ *
+ * @param {Function} model
+ */
+Schema.prototype.loadClass = function(model, virtualsOnly) {
+  if (model === Object.prototype || model === Function.prototype) {
+    return this;
+  }
+
+  // Add static methods
+  if (!virtualsOnly) {
```

Comment:
Hmm what's the point of this virtualsOnly option? Looks like it's always true anyway...